hydra/test_utils/launcher_common_tests.py

```diff
@@ -346,6 +349,15 @@ def sweep_two_config_groups(
 
 @pytest.mark.usefixtures("hydra_restore_singletons")
 class IntegrationTestSuite:
+
+    def post_run_callback(
+        self, **kwargs: Any
+    ) -> Optional[Callable[[Dict[Any, Any]], None]]:
+        """
+        This will be called after running test app and before validating output file from test.
```

Comment:
You can give some context in the comment:
"Typical use case is to download remote working directory locally to allow for validation of remotely launched jobs".

Reply:
added.